snippets/modules.json

```diff
@@ -2239,7 +2239,7 @@
 		"body": ["memset(${1:dst}, ${2:c}, ${3:count})"]},
 	"oledll Windows only: Creates () from the ctypes module": {
 		"description": "oledll Windows only: Creates () from the ctypes module",
-		"prefix": "oledll Windows only: Creates ",
+		"prefix": "oledll Windows only; Creates ",
```

Comment:
what does this fix @obayit

Reply:
So I just found this page from looking up "oledll Windows only: Creates ()" because every time I hit : I am getting these suggestions.
![image](https://user-images.githubusercontent.com/4523585/137646064-81ea22e3-365d-46e5-9be2-5cdc566378d5.png)

It would close #8 apparently.